src/implementations/twig/components/site-header-harmonised/site-header-harmonised.story.js

```diff
@@ -33,6 +34,9 @@ const dataG1 = { ...dataGroup1 };
 const dataG2 = { ...dataGroup2 };
 const dataG3 = { ...dataGroup3 };
 
+const system = process.env.STORYBOOK_SYSTEM === 'EU';
```

Comment:
that's a detail, but `system` is a boolean here, so maybe the name could be changed to something more explicit (`isSystemEU` ?)